Gemfile.lock

```diff
@@ -107,6 +107,7 @@ GEM
     backports (3.8.0)
     bcrypt (3.1.11)
     bindata (2.4.0)
+    bluetooth (1.0)
```

Comment:
Building with `DevKit` seems to be failing, but i'm 95% sure that's because my windows computer doesn't have a bluetooth adapter (and has never installed the drivers for one)